src/components/number/number.test.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { createRef } from 'react';
+import {
+  render,
+  screen,
+  cleanup,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Number from './number.jsx';
+
+describe('Number', () => {
+  const requiredProps = {
+    id: 'test-component',
+    label: 'Test Component',
+  };
+
+  afterEach(cleanup);
+
+  it('has defaults', () => {
+    const ref = createRef();
+    render(<Number {...requiredProps} ref={ref} />);
+    expect(document.body).toMatchSnapshot();
+    expect(ref.current).toMatchSnapshot();
+  });
+
+  describe('disabled API', () => {
```

Comment:
While we are moving away from `disabled`, this test coverage is incomplete. When we are testing a boolean, we need to test the `true` and `false` cases. Since it's only two possible states, it's reasonable to get an exhaustive test suite on it.